doc/UsersGuide/source/ContributingDevelopment.rst

```diff
@@ -13,4 +13,284 @@ Contributing Development
 * What are the requirements for a code to be accepted?
 
 * What platforms are supported
+
+================================
+ ufs-community/ufs-weather-model
+================================
+
+------------------------------
```

Comment:
Before diving into the technical aspects, please include one or more introductory paragraph(s) explaining the big picture of how the community can contribute development.

Reply:
The ufs-weather-model repository contains the model code and external links needed to build the Unified Forecast System (UFS) atmosphere model and associated components, including the WaveWatch III model. This weather model is used in several of the UFS applications, including the medium-range weather application, the short-range weather application, and the sub-seasonal to seasonal application.